updates.js

```diff
@@ -0,0 +1,12 @@
+const completion = (e) => {
```

Comment:
- Great job implementing this function💯. But due to naming and readability purposes, the function name `completion` is not very descriptive of what the function does. A better name could be `handleTaskCompletion`.